edx_name_affirmation/models.py

```diff
@@ -11,6 +12,32 @@
 User = get_user_model()
 
 
+class VerifiedNameStatus(str, Enum):
```

Comment:
Can we move this class into a new file called `status.py`? That sets up a better convention precedence for other engineers to follow

Reply:
a4c3ee6 - chose statuses.py to match proctoring and the general django pattern of plurals - models, views, statuses